packages/grafana-ui/src/components/Logs/LogMessageAnsi.tsx

```diff
@@ -1,8 +1,9 @@
-import { findHighlightChunksInText } from '@grafana/data';
+import { findHighlightChunksInText, GrafanaTheme2 } from '@grafana/data';
 import ansicolor from 'ansicolor';
 import React, { PureComponent } from 'react';
 // @ts-ignore
 import Highlighter from 'react-highlight-words';
+import { Themeable2, withTheme2 } from 'src';
```

Comment:
i think most of frontend test are failing because of this:

```suggestion
import { Themeable2 } from '../../types';
import { withTheme2 } from '../../themes';
```

Reply:
Apologies for the delay @Elfo404 - I've been out sick all week. I've pushed some changes to (1) fix the existing tests (including this import), and (2) to add a new test to ensure ANSI dim is rendered with the correct theme color.